src/Raven.Client/Data/DatabaseStatistics.cs

```diff
@@ -79,6 +79,8 @@ public class IndexInformation
         public IndexType Type { get; set; }
 
         public DateTime? LastIndexingTime { get; set; }
+
+        public string OutputReduceToCollection { get; set; }
```

Comment:
For debugging, we used to get all info about an index when going to the /stats so it made sense to me to add it here. And in order to keep it thin I added it only if it isn't null.